doc/03_reference/cs_registers.rst

```diff
@@ -576,6 +582,33 @@ The User Mode ``time(h)`` registers are not implemented in Ibex.
 Any access to these registers will trap.
 It is recommended that trap handler software provides a means of accessing platform-defined ``mtime(h)`` timers where available.
 
+Machine Vendor ID (mvendorid)
+-----------------------------
+
+CSR Address: ``0xF11``
+
+Reset Value: ``0x0000_0000``
+
+Use the ``MVENDORID_VALUE`` localparam in :file:`rtl/ibex_cs_registers.sv` to change the fixed value.
+
+Machine Architecture ID (marchid)
+---------------------------------
+
+CSR Address: ``0xF12``
+
+Reset Value: ``0x0000_0016``
+
+Use the ``MARCHID_VALUE`` localparam in :file:`rtl/ibex_cs_registers.sv` to change the fixed value.
```

Comment:
Maybe add a note that these numbers are assigned to us, and people should change them if they make significant changes (can we be more specific?) to the core?